### ISSUE TITLE
Redesign the CONTRIBUTING.md file

### ISSUE THREAD

**Issue by Niweera** [Niweera/cinesimile#7]
If you are a first-time contributor you can contribute to this project by re-designing the [CONTRIBUTING.md](https://github.com/Niweera/cinesimile/blob/master/CONTRIBUTING.md) file. Use your imagination and make it more beautiful. However, please do not remove important information from the [CONTRIBUTING.md](https://github.com/Niweera/cinesimile/blob/master/CONTRIBUTING.md) file. Feel free to add more information as you see fit.

**Comment by neel229** [comment]
Can I take this issue?

**Comment by Niweera** [comment]
@neel229, yes sure you can. Thank you for your contribution to the project. Happy Hacktoberfest! 😁

**Comment by neel229** [comment]
Okay @Niweera

**Comment by sakshi006** [comment]
Hey!
Can i work on this issue?
This is the first time i will be contributing. Please help.

**Comment by Niweera** [comment]
Actually there is another person assigned for this. I'll let you know once I add a new task.

**Comment by sakshi006** [comment]
@Niweera  Sure. Thankyou. :)

**Comment by neel229** [comment]
Actually @sakshi006 you can work on it. I'm thinking of making PRs related to coding in Golang. @Niweera you can assign the task to @sakshi006 .

**Comment by Niweera** [comment]
In that case @sakshi006 can work on this.
I'll assign @sakshi006 and unassign @neel229

**Comment by sakshi006** [comment]
Thanks.

**Comment by prabhu4444** [comment]
I'd like to contribute, can I work on this issue?

**Comment by Niweera** [comment]
@prabhu4444 yes you can! Go do something wonderful.
